data_steward/gcloud/gcs/__init__.py

```diff
@@ -129,3 +123,7 @@ def list_sub_prefixes(self, bucket: str, prefix: str) -> None:
             extra_params=extra_params,
         )
         return list(pg_iterator)
+
+    def _get_hpo_bucket_id(self, hpo_id: str) -> str:
```

Comment:
I'd suggest `_get_hpo_bucket_name` since you're using the string `bucket_name` in usages above.
```suggestion
    def _get_hpo_bucket_name(self, hpo_id: str) -> str:
```

Reply:
Committed.  I agree these are better names.